test/support/api/clinicians.js

```diff
@@ -1,16 +1,28 @@
 import _ from 'underscore';
+import moment from 'moment';
 import { getResource, getIncluded, getRelationship } from 'helpers/json-api';
 
 Cypress.Commands.add('routeCurrentClinician', (mutator = _.identity) => {
   cy
+    .fixture('collections/groups').as('fxGroups')
     .fixture('test/clinicians').as('fxTestClinicians');
 
   cy.route({
     url: '/api/clinicians/me',
     response() {
+      const groups = _.sample(this.fxGroups, 2);
```

Comment:
this seems moderately problematic... maybe not, but the current clinicians is creating 2 groups, ensuring that the current clinician is in both groups.  We _probably_ need to hard set the group ids here and there to both be `11111` and `22222` and _not_ include the group data here.

Reply:
The main issue I was trying to tackle here was guaranteeing that the currentClinician was active in all tests. I think the original issue was that, even though `routeGroupsBootstrap` sets the groups for all clinicians, if an individual test called `routeCurrentClinician`, it would override the data set by `routeGroupsBootstrap`, causing tests to get stuck on the "account not set up" view.

Hardcoding the group ids like you suggested works just as well, though.